src/syntax.ml

```diff
@@ -685,7 +686,8 @@ let rec block_element_list :
                 let tag =
                   match tag with
                   | `Param s -> `Param (s, content)
-                  | `Raise s -> `Raise (s, content)
+                  | `Raise (`Word s) -> `Raise (`Word s, content)
+                  | `Raise (`Simple_reference s) -> `Raise (`Reference (`Simple, Loc.at location s, []), content) (* TODO: only s location *)
```

Comment:
I'm not sure how to get a location for just the simple reference here when it's lexed inside a raise tag.